native/src/components/CalendarChoice.tsx

```diff
@@ -1,28 +1,23 @@
-import { HeaderBackButton } from '@react-navigation/elements'
 import React, { ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
 import { Modal, TouchableOpacity } from 'react-native'
 import { Calendar } from 'react-native-calendar-events'
 import styled from 'styled-components/native'
 
-const StyledView = styled.View`
+import dimensions from '../constants/dimensions'
```

Comment:
Sorry for not mentioning this earlier but I think we should somehow make clear that this is a full screen modal/overlay and therefore name it `CalendarChoiceModal` or `CalendarChoiceOverlay`. Since react-native seems to use the term modal and we do in lunes as well, I would prefer the first choice. I would then also be consistent and avoid the term overlay (e.g. in ExportEventButton).